projects/frontend-ui/shared/projects/gui/proxy.config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "/metadata/lineage/datajob:test_datajob": {
```

Comment:
metadata" should not be a part of the PR

Reply:
Will I delete the entire project.config.json file?